.github/ISSUE_TEMPLATE

```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
 If it's a bug please describe what was the unexpected thing that occured and what was the
 expected behaviour.
 
-## Solution
-
-Here provide a detailed explanation, if you can suggest one, for how to go about solving the problem.
-
-
-## Tasklist
-
-Here provide a checkpoint list of tasks that need to be done in order for this issue to be considered closed.
-This will help us evaluate if the issue has been properly addressed.
-
-Example:
-
-- [ ] Implement feature XXX
-- [ ] Create test for XX
-- [ ] Add a changelog entry.
-
-## Only For Bugs
+### System Description
 
-Bother with this section only if you are reporting a bug
+Here add a detailed description of your system, e.g. output of the following script:
 
-### System Description
-Here add a detailed description of your system, e.g. output of `raiden version`, `geth version`, `solc --version` e.t.c.
+```
+uname -a
+command -v solc && solc --version
+command -v geth && geth version
+command -v parity && partiy --version
+command -v raiden && raiden version
```

Comment:
perhaps mention here if it's built from source show the hash of the commit? like `git show HEAD`?

Reply:
```
raiden version
{
  "raiden": "0.2.0+git.r129fa28a",
  "python_implementation": "CPython",
  "python_version": "3.6.4",
  "system": "Linux 64bit_ 4.14.14-1-ARCH x86_64"
}
```

Reply:
Hmm good point but I meant it also as an alternative in the case they have not managed to make raiden run, so `raiden version` would not work.